### ISSUE TITLE
Multiple incoherences

### ISSUE THREAD

**Issue by manel00** [explosion/spaCy#5529]
Hello, 

First of all thanks for such nice job, but i would like to post some text which tags arent correctly:

Trato LOC (Deal)
Flexibilidad ORG (Flexibility)
Mal LOC (Bad)
Nissan ORG
Nissan LOC

Cannot understand why in just the same text are different TAGS for Nissan, and so on? Maybe also the other ones are because for spanish the training it doesnt work properly?

Thanks in advance

* Operating System: W10
* Python Version Used: 3.8
* spaCy Version Used: 2.2.4

**Comment by svlandeg** [comment]
The entity type or label depends on the sentence (context) in which an entity is expressed. Theoretically, it may well be that "NISSAN" is both a location somewhere, as well as an organization.

That said, it may well be that the Named Entity Recognizer makes mistakes, as no Machine Learning model is perfect. Additionally, the pretrained [models](https://spacy.io/models) are not necessarily a good fit for your use-case. If you have texts that are quite different than the training texts, the model may not perform well. In this case, you probably want to generate some annotated training data and [train your own NER model](https://spacy.io/usage/training#ner) on your data.
